vcpkg.json

```diff
@@ -160,10 +169,6 @@
       "name": "trantor",
       "version": "1.5.11#2"
     },
-    { 
```

Comment:
You don't need this override at all now? I noticed you're not overriding it in rocal's vcpkg.json either.